### PR TITLE
feat: added alert component

### LINKED ISSUE #8: Create alert box
Need alert box for multiple use cases like success, error, info and warning.

### PR DESCRIPTION
**Pull request by Vivek-Py**
### Summary

Added alert component

### Checks

- [X] Demo added in the index file.
- [X] Sample screenshot added.

### Preview

image

### CODE REVIEW

File: components/alert/alert.css [hunk 1]
```diff
@@ -0,0 +1,23 @@
+[class*="alert"] {
+  box-sizing: border-box;
+  border: none;
+  border-radius: 0.3rem;
```
Comment (aurobindoGupta) [inline components/alert/alert.css:4]: maybe use px for really small values

File: components/alert/alert.css [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  padding: 1rem;
+  margin: 1rem;
+}
+
+.alert--success {
+  background: #d4edda;
```
Comment (aurobindoGupta) [inline components/alert/alert.css:10]: go for color variables, make a different file for colors and import here, i was thinking of using it that way

**Review by aurobindoGupta — APPROVED** [review]
nice work

File: components/alert/alert.css (hunk 2 above)
Comment (Vivek-Py) [reply components/alert/alert.css:10]: Good suggestion, thanks!

File: components/alert/alert.css (hunk 1 above)
Comment (Vivek-Py) [reply components/alert/alert.css:4]: Alrighty!

**Review by tanishk15sharma — COMMENTED** [review]
clean code

**Pull request merged**